src/main/java/org/candlepin/async/tasks/InactiveConsumerCleanerJob.java

```diff
@@ -141,6 +141,7 @@ private Integer deleteInactiveConsumers(Object... args) {
         List<String> scaCertsToRemove = consumerCurator.getContentAccessCertIds(consumerIds);
         List<Long> serialIdsToRevoke = this.consumerCurator.getConsumerCertSerialIds(consumerIds);
 
+        this.deletedConsumerCurator.deleteByConsumerIds(consumerIds);
         this.deletedConsumerCurator.createDeletedConsumers(consumerIds);
         int deletedConsumers = this.consumerCurator.deleteConsumers(consumerIds);
```

Comment:
I think something got lost from the last review. These three lines should become a single line:
```
int deletedConsumers = this.consumerCurator.deleteConsumers(consumerIds);
```

The management of the deleted consumers entries is an implementation detail of the delete operation. Callers should not be expected to have to know how to manage that.

Reply:
Good idea, I also move deleting of the `DeletedConsumer` to createDeletedConsumer method. Initialy that move seems wrong to me but it is how our creation should work so move it there. If you think it is wrong I will move it to delete method for consumer.